packages/harmony/src/components/layout/Paper/Paper.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled'
+
+import { toCSSVariableName } from 'utils/styles'
+
+import type { PaperProps } from './types'
+
+/** Base layout component used as a building block for creating pages and other components. */
+export const Paper = styled.div(
+  ({
+    h,
+    w,
+    backgroundColor = 'default',
+    border = 'default',
+    borderRadius = 'm',
+    shadow = 'mid'
+  }: PaperProps) => {
+    return {
+      position: 'relative',
+      boxSizing: 'border-box',
+      height: h,
+      width: w,
+      boxShadow: shadow && `var(--harmony-shadow-${shadow})`,
```

Comment:
If you're down, you should be able to use the theme object now

Reply:
yeah I'll circle back tomorrow morning and use the new theme stuff

Reply:
Updated! I think I did it right, but if you want, feel free to go through it again.